Automattic/wp-calypso
# Cloud Based: Remove Unused Illustration

#### Changes proposed in this Pull Request

As mentioned [here](https://github.com/Automattic/wp-calypso/issues/36850#issuecomment-596816734), unused illustrations should be removed as part of #36850, though they could always be recovered in the future. This is one of them.

#### Testing instructions

Verify there's no uses of this illustration: https://github.com/Automattic/wp-calypso/search?q=%2Fcalypso%2Fimages%2Fillustrations%2FcloudBased.svg&unscoped_q=%2Fcalypso%2Fimages%2Fillustrations%2FcloudBased.svg
